watsonsdk/OpusHelper.h

```diff
@@ -0,0 +1,122 @@
+/**
+ * Copyright IBM Corporation 2015
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ **/
+
+#import <Foundation/Foundation.h>
+#import <AVFoundation/AVFoundation.h>
+#import "CodecHeader.h"
```

Comment:
AFAIK OpusHelper.h is a public header. importing CodecHeader.h in a public header will required clients to import all headers inside and most of them are not public. Could this header to the .m file please? To do so all ogg type mentions will be need to be done in the .m file as class extension